test/lit/extern-conversions.wast

```diff
@@ -0,0 +1,33 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
```

Comment:
This file will need to be added to `fuzz_opt`'s ignore-list, like `strings.wast`, until the interpreter can execute this stuff.